datavault-webapp/src/main/java/org/datavaultplatform/webapp/controllers/admin/AdminRolesController.java

```diff
@@ -205,4 +289,19 @@ public ResponseEntity deleteRole(@RequestParam("id") long roleId) {
         List<PermissionModel> permissions = restService.getSchoolPermissions();
         return ResponseEntity.ok(permissions);
     }
+
+    private boolean isIsAdmin(Principal principal) {
```

Comment:
This really needs a better name. How about `isSuperAdmin`, or `isGlobalAdmin`? Unless you know what the `IS` part stands for and we can expand it out.

Reply:
renamed to SuperAdmin where possible